### ISSUE TITLE
Unable to save file inside GNU Nano

### ISSUE THREAD

**Issue by papycochon** [fnuecke/oc2#51]
The enter (RETURN) is considerated to be ^J (CTRL+J) inside GNU NANO, causing the issue of not being able to save the file.
![image](https://user-images.githubusercontent.com/64284419/126621368-b293b654-5c8f-4880-9ef5-527bfa97eaaa.png)

(+ you can't make a newline without pressing the arrow down button, thanks to CRLF)

**Comment by papycochon** [comment]
I guess I'll use vi

**Comment by CoolCat467** [comment]
That sounds like keys being bound improperly

**Comment by papycochon** [comment]
> That sounds like keys being bound improperly

Yup, but that only happens inside nano. At first I thought it could be due to my keyboard locale but now I think it's not that (fr-FR)

**Comment by fnuecke** [comment]
Yeah, known issue, probably due to the very primitive terminal emulator. Likely to be related to #33.

**Comment by atirut-w** [comment]
This issue kind of sitrs up various questions at least for me. Does this mean OC2 will work by connecting to a real computer? Or is it just that the OS included with OC2 will include a sort of SSH program that connects to localhost?

**Comment by RGFTheCoder** [comment]
> This issue kind of stirs up various questions at least for me. Does this mean OC2 will work by connecting to a real computer? Or is it just that the OS included with OC2 will include a sort of SSH program that connects to localhost?
@atirut-w OC2 uses a vm that runs in minecraft; Using a sort of ssh would be very insecure for servers.

**Comment by atirut-w** [comment]
> OC2 uses a vm that runs in minecraft

So OC2 computers run actual machine code instead of Lua?

**Comment by CoolCat467** [comment]
Sort of, they run code in a virtual machine (vm) that runs on top of Java that runs at the same time as minecraft. If you have a multi-player server and have oc2 on it, the server is what actually runs everything. Clients are only the interface to the computers, ie. almost nothing is processed on the client side for the computers except rendering and talking to the server for security reasons.

**Comment by atirut-w** [comment]
> Sort of

The hell do I make my own OS now lmao
